nvm.sh

```diff
@@ -337,8 +337,8 @@ nvm_num_version_groups() {
     echo "0"
     return
   fi
-  VERSION="${VERSION#v*}"
-  VERSION="${VERSION%\.}"
+  VERSION="${VERSION#v}"
+  VERSION="${VERSION%.}"
```

Comment:
I've made a functionality change here so that "v" and "." report 0 in c34502ebc2cf4fe5ea8c3b0ef8d21807f092173a, and incorporated these changes there. Apologies for the merge conflict :-/ a fresh rebase should sort it?